### ISSUE TITLE
Make an Icon

### ISSUE THREAD

**Issue by AbhinavMadahar** [Charles-T-King/website#4]
We currently do not have an icon for Charles T. King because they don't have one. That means that we have to create an icon ourselves. Please add your ideas or submissions in the comments below.

**Comment by AbhinavMadahar** [comment]
@16yildiza do you want to make the icon?

**Comment by 16yildiza** [comment]
YES YES YES YES YES! OF COURSE!

**Comment by 16yildiza** [comment]
I very much would love to make an icon. Do they have anything they would like to be included in the logo? Like a theme, mood, feel, colors, circular vs. flat, etc? I would be happy to design a logo.

**Comment by AbhinavMadahar** [comment]
@16yildiza something professional. Preferably flat and non-circular because that seems to be how most websites are going nowadays. I think you're the best decider of the specifics.

**Comment by 16yildiza** [comment]
So... I'm guessing the color of the logo will depend on the theme? I could color it many different ways and then as a group we can be the final deciders. However, if there are color specifics from the organization I would love to know since they would like it more.

**Comment by AbhinavMadahar** [comment]
@16yildiza we can probably decide on the colours tomorrow during class. Thanks for making it!

**Comment by shannonpeng** [comment]
@16yildiza If you're drawing it can I trace over it in Illustrator? :D

**Comment by 16yildiza** [comment]
Oh @shannonpeng that would be amazing! But you don't have to.
@AbhinavMadahar So you want it for tomorrow? If so, I'll have some design ideas by then, but they would not digitally rendered/edited/colored. I don't have the time today because I am studying for the AP Macro qualifier.

**Comment by 16yildiza** [comment]
I designed one idea! It looks pretty good!

**Comment by AbhinavMadahar** [comment]
@16yildiza you can take your time

**Comment by shannonpeng** [comment]
@16yildiza Yay!

**Comment by shannonpeng** [comment]
![screen shot 2015-03-17 at 5 51 36 pm](https://cloud.githubusercontent.com/assets/10291202/6698373/70573c7a-ccce-11e4-8920-977fe3995f44.png)
As minimal as possible

**Comment by AbhinavMadahar** [comment]
That's awesome! Can you make a (square) favicon?

**Comment by shannonpeng** [comment]
![charles-t-king-icon](https://cloud.githubusercontent.com/assets/10291202/6699579/aad88d12-ccd9-11e4-9a6c-be1a7b1f2e8a.png)

**Comment by shannonpeng** [comment]
(I promise it's square but the cap doesn't take up the entire thing)

**Comment by 16yildiza** [comment]
Nice job Shannon! We might need to change the colors thought, if the non-profit wants otherwise.

**Comment by 16yildiza** [comment]
![charles](https://cloud.githubusercontent.com/assets/11217269/6700509/715d140c-cce5-11e4-884c-b4d99f855c96.png)
This is another tentative idea!

**Comment by 16yildiza** [comment]
@AbhinavMadahar @shannonpeng Shannon and I like this one the best!
![charles 2](https://cloud.githubusercontent.com/assets/11217269/6700725/7996d448-cce8-11e4-836b-af750853747f.jpg)

**Comment by shannonpeng** [comment]
@AbhinavMadahar @16yildiza here's the full palette: 
![screen shot 2015-03-17 at 9 06 05 pm](https://cloud.githubusercontent.com/assets/10291202/6700794/86bc1ba0-cce9-11e4-999d-24ab36c54b82.png)

**Comment by shannonpeng** [comment]
I think tomorrow we should write "CTK" in perspective across the top of the hat so we can use that as an icon in itself as well.

**Comment by shannonpeng** [comment]
And maybe we'll write "student loan fund" in white across the banner

**Comment by shannonpeng** [comment]
I still don't want to give up the crown though. We should incorporate that somewhere.

**Comment by 16yildiza** [comment]
@shannonpeng what we could do is make the banner bigger and put the "student loan fund" phrase so it's legible. And the perspective thing is a great idea!
Maybe we can have a little yellow crown as a period! :D

**Comment by AbhinavMadahar** [comment]
I think the first one is the best since it's flat and more professional, whereas the rest (although spectacular) are more playful, and we want to be taken seriously.

![My favourite](https://cloud.githubusercontent.com/assets/10291202/6698373/70573c7a-ccce-11e4-8920-977fe3995f44.png)

**Comment by 16yildiza** [comment]
That makes sense. I think all of them are amazing. Serious vs. playful. I also think we need to have a logo that's also inviting. You know what I mean?
Also, I just looked online to see if these logo ideas were taken, and many of them tend to have a graduation hat. But the good news is that they aren't in the way we have presented.
Btw, do we need anything else for the website as in designs?
Another suggestion, @shannonpeng I think we should make the T in the flatter version bigger, because it looks like it is hiding under the hat (looks smaller as a result). Only a suggestion. It still looks amazing.

**Comment by 16yildiza** [comment]
I WANT TO CONGRATULATE EVERYONE FOR ALL THEIR HARD WORK! We made so many different versions of the same name "Charles T. King" and @shannonpeng, they all look amazing. I hope the company will be pleased.

**Comment by serialscoper101** [comment]
Sweet. I'll tell Conor about our progress so far.  Maybe he or the organization have any other ideas/suggestions.

**Comment by 16yildiza** [comment]
@serialscoper101 that would be amazing! If he needs different colors, fonts, etc, we are on it!

And thanks for the promotion Abhinav and Rus! I will do my best to help design the website. @shannonpeng is the ultimate assistant.

**Comment by serialscoper101** [comment]
Take a look at [meetings and resources so far](https://drive.google.com/open?id=0BxEuMs2jIxWfV0VNQ1ZzZFJpUkU&authuser=0). Maybe it will be useful

**Comment by 16yildiza** [comment]
@serialscoper101 Will do!

**Comment by shannonpeng** [comment]
Sounds great :) So what are we focusing on tomorrow?

**Comment by serialscoper101** [comment]
No specific focus but continue working on the codecademy tutorials if you haven't finished them.  Mr. Farrell will be absent Thursday and Friday;  there will be no laptops so I don't expect anything to get done. But by the end of this week, I would love if everyone knew their position on weather they would like to pursue backend or frontend development. Or both.

**Comment by 16yildiza** [comment]
![charles t king logo](https://cloud.githubusercontent.com/assets/11217269/6716177/ee0876b0-cd79-11e4-95e6-606ef0ec5a36.jpg)

**Comment by serialscoper101** [comment]
Can we try the font Barkentina?

**Comment by shannonpeng** [comment]
![screen shot 2015-03-18 at 6 10 54 pm](https://cloud.githubusercontent.com/assets/10291202/6720542/22767580-cd9a-11e4-90f8-11083d9997fd.png)

**Comment by shannonpeng** [comment]
Is there a font that looks exactly like this but with the gaps in the a, e, d, and g connected?

**Comment by serialscoper101** [comment]
I don't think so but it's possible to edit the individual characters in Illustrator I belive.

**Comment by AbhinavMadahar** [comment]
**This one looks really good!**

![](https://cloud.githubusercontent.com/assets/10291202/6720542/22767580-cd9a-11e4-90f8-11083d9997fd.png)

**Comment by AbhinavMadahar** [comment]
Can you make the background on the logo transparent? The logo will be put against a gray background, so it looks strange.

**Comment by 16yildiza** [comment]
Oooooh I like this one as well! There are so many choices!

**Comment by serialscoper101** [comment]
no need to export a transparent png for now.

**Comment by 16yildiza** [comment]
Agreed. We still need Conor and Charles T. King to decide which logo to use.

**Comment by shannonpeng** [comment]
I'll change the letters in Illustrator and/or get you guys the transparent png as soon as we get their choice on which logo to use. :)

**Comment by shannonpeng** [comment]
After rethinking I decided a serif font would look good for the body of the site - this font looks pretty good (ignore the article).
![image](https://cloud.githubusercontent.com/assets/10291202/6738028/b742d0e0-ce45-11e4-94af-bde2a8c4ec5f.jpg)

**Comment by AbhinavMadahar** [comment]
So what font do you want?

**Comment by shannonpeng** [comment]
They used Utopia Std (https://typekit.com/fonts/utopia-std), but we don't
have to use that exact font.

**Comment by AbhinavMadahar** [comment]
@shannonpeng I found a [cool list](http://www.awwwards.com/20-best-web-fonts-from-google-web-fonts-and-font-face.html) of web fonts we can use. You don't have to use them, but they're still all good in their own way.

**Comment by serialscoper101** [comment]
We will use the fonts available on google fonts.

**Comment by AbhinavMadahar** [comment]
@serialscoper101 the list is of Google Web Fonts, so we can use them

**Comment by serialscoper101** [comment]
Sure, I didn't read the article

**Comment by AbhinavMadahar** [comment]
I made the icon for our GitHub Organization the little hat that you made a while ago. We can change later, but I think it looks nice.

**Comment by AbhinavMadahar** [comment]
I will close this issue now given that we seem to have mostly agreed upon a few designs. Later on, we'll talk to the Charles T. King people and see which designs they like.

If anyone thinks this issue should be open, please reopen it.

**Comment by serialscoper101** [comment]
We will need 2 more different logo variations to present to Charles T. King on Friday.

**Comment by serialscoper101** [comment]
Also can we try that logo but without "the" on the left side?  Thanks

**Comment by AbhinavMadahar** [comment]
@serialscoper101 you mean the first one? ![First](https://cloud.githubusercontent.com/assets/10291202/6698373/70573c7a-ccce-11e4-8920-977fe3995f44.png)

**Comment by serialscoper101** [comment]
Yes.

**Comment by AbhinavMadahar** [comment]
Sorry, I deleted my last message. Here is what it was- ![First](https://cloud.githubusercontent.com/assets/10291202/6698373/70573c7a-ccce-11e4-8920-977fe3995f44.png)

**Comment by shannonpeng** [comment]
In the .ai file in the repo, I have an updated version of the first logo
but with "the" moved above "Charles". Is that fine or would it be better to
remove "the" entirely?
